--- a/verilog/providers.bzl
+++ b/verilog/providers.bzl
@@ -124,11 +124,14 @@ verilog_library = rule(
         ),
         "hdrs": attr.label_list(
             doc = "Verilog or SystemVerilog headers.",
-            allow_files = [".vh", ".svh"],
+            # Some vendor libraries include .h, .v, and .sv files rather than
+            # following the .vh/.svh convention.
+            allow_files = [".vh", ".svh", ".h", ".v", ".sv"],
         ),
         "srcs": attr.label_list(
             doc = "Verilog or SystemVerilog sources.",
-            allow_files = [".v", ".sv"],
+            # .vp and .svp are encrypted files.
+            allow_files = [".v", ".sv", ".vp", ".svp"],
         ),
     },
 )
