--- a/src/mod_def/parameterize.rs
+++ b/src/mod_def/parameterize.rs
@@ -98,7 +98,7 @@ impl ModDef {
             .map(|(name, value)| {
                 // Get the width from the bigint itself, not from the parameter definition.
                 // TODO(sherbst) 2025-10-29: Support negative parameter values
-                let width = value.bits();
+                let width = value.bits().max(1);
                 let str_value = match value.sign() {
                     Sign::Plus | Sign::NoSign => format!("{width}'d{value}"),
                     Sign::Minus => panic!("Negative parameter values not yet supported"),
